guides/common/modules/snip_resource-remove-vm.adoc

```diff
@@ -0,0 +1,2 @@
+* You can configure {Project} to remove the associated virtual machine when you delete a host.
+For more information, see xref:removing-a-virtual-machine-upon-host-deletion_provisioning[].
```

Comment:
```suggestion
For more information, see xref:removing-a-virtual-machine-upon-host-deletion_{context}[].
```

IMHO we only have "{DocURL..}..._hardcoded_context" or "xref:..._{context}". Is this necessary because we overwrite "{context}" for individual compute resources? If so, I small comment would be helpful to (future) me.

Reply:
Yes, that's exactly why this is necessary.

Reply:
Same! :)